annif/rest.py

```diff
@@ -50,6 +51,18 @@ def show_project(project_id):
     return project.dump()
 
 
+def detect_language(body):
+    """return scores for detected languages formatted according to Swagger spec"""
+
+    scores = lang_detector(body.get("text"), tuple(body.get("candidates")))
+    return {
+        "results": [
+            {"language": s[0] if s[0] != "unk" else None, "score": s[1]}
```

Comment:
How about this instead:

```python
            {"language": lang if lang != "unk" else None, "score": score} for lang, score in scores
```

I think it would be easier to understand than the numeric indexing.